src/osw_config_keys.cpp

```diff
@@ -29,6 +29,7 @@ OswConfigKeyBool lightSleepEnabled("s7", "Energy Settings", "Light Sleep", "Use
                                    false);
 OswConfigKeyBool tapToWakeEnabled("s8", "Energy Settings", "Tap to Wake",
                                   "Enables Tap to Wake (If you select none, button 1 will wake the watch)", true);
+OswConfigKeyBool buttonToWakeEnabled("m","Energy Settings","Button to Wake","Enables Button to wake",SLEEP_BUTTONWAKE);
```

Comment:
Also some line formatting here (spaces).

Reply:
trying the --amend lets hopefully i dont botch it!